_posts/2022-04-08-introducing-device-groups.markdown

```diff
@@ -0,0 +1,399 @@
+---
+layout: post
+title:  "Introduction to edge device groups"
+date:   2022-04-08 12:00:00 +0100
+categories: flotta
+author: Jakub Dżon
+tags:
+  - guide
+  - flotta
+  
+summary-1: Introduction to edge device groups
+---
+
+If you are Flotta user that needs to manage large numbers of devices and doesn't want to change configuration of each 
+of them separately, using Edge Device Groups is the perfect approach. With the use of Edge Device Groups you can change 
+configuration of any number of device by changing ony one CR - `EdgeDeviceGroup` and this blog post will show you how.
+
+### EdgeDeviceGroup configuration
+
+User may define the same configuration elements using `EdgeDeviceGroup` as one would with `EdgeDevice` and configuration 
+defined in the `EdgeDeviceGroup` would be applied to all edge devices using it. `EdgeDeviceGroup`  **always** takes 
+precedence over whatever is defined in the `EdgeDevice`as a whole. It means that even if some element is not present in 
```

Comment:
I think there is no space between "EdgeDevice" and "as"

Reply:
Indeed. Done.